src/main/java/pt/dinis/main/App.java

```diff
@@ -1,9 +1,24 @@
 package pt.dinis.main;
 
+import pt.dinis.temporary.Configurations;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
 public class App {
-    public static void main( String[] args ) {
-		Dealer dealer = new Dealer();
-		dealer.start();
-    }
+    public static void main( String[] args ) throws IOException {
+        Dealer dealer;
 
+        try {
+            Configurations.setPropertiesFromFile();
+        } catch (FileNotFoundException e) {
+            Display.alert("Important files not found, this program will close now.");
```

Comment:
Estou a optar por mensagens mais curtas: Properties not found
E depois mandar para o logger mais info